Doc/reference/lexical_analysis.rst

```diff
@@ -503,7 +503,8 @@ must be expressed with escapes.
    single: r"; raw string literal
 
 Both string and bytes literals may optionally be prefixed with a letter ``'r'``
-or ``'R'``; such strings are called :dfn:`raw strings` and treat backslashes as
+or ``'R'``; such objects are called :dfn:`raw strings`
+and :dfn:`raw bytes` respectively and treat backslashes as
```

Comment:
Later on (the 3.3 versionadded note) we use the phrase "raw bytes literals", which I think reads better.

```suggestion
and :dfn:`raw bytes literals` respectively and treat backslashes as
```

Reply:
@terryjreedy also pointed this out. I think that I will use this term. Thanks!